app/js/components/StatusContactCode.jsx

```diff
@@ -19,15 +20,28 @@ const StatusContactCode = ({t, value}) => (
     <InputGroup className="full-width-input white-input">
       <Input type="text" name="contactCode" className="prepend" disabled defaultValue={value}/>
       <InputGroupAddon addonType="append">
-        <InputGroupText><FontAwesomeIcon size="lg" icon={faQrcode}/></InputGroupText>
+        <InputGroupText>
+          <Button onClick={toggleQRCode}>
+            <FontAwesomeIcon size="lg" icon={faQrcode}/>
+          </Button>
+        </InputGroupText>
       </InputGroupAddon>
     </InputGroup>
+
+    {showQRCode && <Row className="border rounded py-4 m-0 shadow-sm">
```

Comment:
I don't know how it looks but a modal maybe?

Reply:
It looks pretty ;)